frontend/src/components/layout/Sidebar.tsx

```diff
@@ -0,0 +1,179 @@
+              {!isCollapsed && <span>{t('sidebar.my_courses')}</span>}
+            </div>
+            {!isCollapsed && (expandedSections.courses ? <ChevronDown size={16} /> : <ChevronRight size={16} />)}
+          </button>
+          {/* Favorite Courses List */}
+          {!isCollapsed && expandedSections.courses && (
+
+              <div>
+                <ItemList
+                    items={mapCoursesToItems(user!.favoriteCourses!)}
+                    emptyMessage={t('account.favorite.no_courses')}
+                    updateFavorite={updateFavoriteCourse}
+                />
+              </div>
+
+          )}
+          <button
+              className="flex items-center justify-between w-full text-gray-700 hover:bg-gray-100 transition-all duration-200 hover:scale-[1.01] rounded p-2"
+              onClick={() => toggleSection('documents')}
+              aria-label={t('sidebar.toggle_documents')}
+          >
+            <div className="flex items-center space-x-2">
+              <File size={20} />
+              {!isCollapsed && <span>{t('sidebar.my_documents')}</span>}
+            </div>
+            {!isCollapsed && (expandedSections.documents ? <ChevronDown size={16} /> : <ChevronRight size={16} />)}
+
+          </button>
+          {/* Favorite Documents List */}
+          {!isCollapsed && expandedSections.documents && (
+              <div>
+                <ItemList
+                    items={mapDocumentsToItems(user!.favoriteDocuments!)}
+                    emptyMessage={t('account.favorite.no_documents')}
+                    updateFavorite={updateFavoriteDocument}
+                />
+              </div>
+          )}
+        </nav>
+
+        {/* Add Document Button */}
+        <button
+
+            className={`mx-4 my-2 flex items-center space-x-2 ${isCollapsed ? 'bg-transparent' : 'bg-indigo-600 hover:bg-indigo-700 transition-all duration-200 hover:scale-[1.01]'} text-white rounded-md py-2 px-4 hover:${isCollapsed ? 'bg-transparent' : 'bg-indigo-700'}`}
+            aria-label={t('sidebar.add_document')}
+            onClick={handleUploadButtonClick}
+        >
+          <Plus size={20} />
+          {!isCollapsed && <span>{t('sidebar.add_document')}</span>}
+        </button>
+
+
+        {/* User Profile - Fixed at Bottom #TODO add avatar input*/}
+        <div className="mt-auto border-t border-gray-200">
+          <div className="p-4 flex items-center space-x-3">
+            <img
+                src={'/images/icons/empty_profile.png'}
+                alt={t('sidebar.user_avatar')}
+                className="w-8 h-8 rounded-full"
+            />
+            {!isCollapsed && <span className="text-sm text-gray-700">{user?.fullName}</span>}
+          </div>
+          {!isCollapsed && (
+              <div className="px-4 pb-2 space-y-2">
+                <a href="vtk.be" className="block text-sm text-gray-600 hover:text-gray-800">
+                  {t('sidebar.go_to_vtk')}
+                </a>
```

Comment:
Use Link instead of a
```suggestion
                <Link href="vtk.be" className="block text-sm text-gray-600 hover:text-gray-800">
                  {t('sidebar.go_to_vtk')}
                </Link>
```